source/stylesheets/_navigation.scss

```diff
@@ -0,0 +1,210 @@
+
+// ==== main nav =====
+.nav-title{
+	color: $text-color-lighter;
+	font-weight: $medium-weight;
+	border-bottom: none;
+	padding: $medium-padding;
+}
+
+ul{
+	list-style-type: none;
+    @extend .center-object;
+}
+
+li {
+	float: left;
+}
+
+li a {
+    display: block;
+    text-align: center;
+    padding-left: 2.3em;
+    padding-right: 2.3em;
+    text-decoration: none;
+    border-bottom: none;
+    font-size: $subhead-size;
+}
+
+// selected navigation bar element 
+.active {
```

Comment:
Classes like these can be tricky, but usually I use them on more of a per-component basis and prefix with an explicit state. For instance:

```CSS
.nav-item {
    ...
    &.is-active {
        ...
    }
}
```